e2e/clients/react/App.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/react-in-jsx-scope */
+import Uppy from '@uppy/core'
+/* eslint-disable-next-line no-unused-vars */
+import React, { useState } from 'react'
+import { Dashboard, DashboardModal, DragDrop } from '@uppy/react'
+import ThumbnailGenerator from '@uppy/thumbnail-generator'
+
+import '@uppy/core/dist/style.css'
+import '@uppy/dashboard/dist/style.css'
+import '@uppy/drag-drop/dist/style.css'
+
+export default App
+
+function App () {
```

Comment:
I find hoisting much harder to read, I don't think we need it here.
```suggestion
export default function App () {
```